src/CreatePage.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { createMovie } from './services/Fetch-utils';
+import { useHistory } from 'react-router-dom';
+
+export default function CreatePage() {
+  const { push } = useHistory();
+  const [title, setTitle] = useState('');
+  const [director, setDirector] = useState('');
+  const [year, setYear] = useState('');
+  const [description, setDescription] = useState('');
+
+  async function handleSubmit(e) {
+    e.preventDefault();
+
+    await createMovie({
+      title: title,
```

Comment:
do these object keys match your supabase table setup? make sure to check the column types as well (bigint v. text or varchar)